test/pylib/manager_client.py

```diff
@@ -118,56 +118,55 @@ async def servers(self) -> List[str]:
         host_list = await self._get_text("/cluster/servers")
         return host_list.split(',')
 
-    async def mark_dirty(self) -> None:
+    async def mark_dirty(self) -> str:
         """Manually mark current cluster dirty.
            To be used when a server was modified outside of this API."""
-        await self._get("/cluster/mark-dirty")
+        return await self._get_text("/cluster/mark-dirty")
 
-    async def server_stop(self, server_id: str) -> bool:
+    async def server_stop(self, server_id: str) -> str:
```

Comment:
Do these `str`s contain any meaningful info? If not, we should just give `None`.

Reply:
Done in latest